src/snowflake/cli/_plugins/streamlit/manager.py

```diff
@@ -57,6 +58,10 @@ def _put_streamlit_files(
         stage_manager = StageManager()
         for file in artifacts:
             if file.is_dir():
+                if not bool(os.listdir(file)):
```

Comment:
```suggestion
                if not any(file.iterdir()):
```
To avoid additional `os` import